bin/webpacker

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env ruby
```

Comment:
@jcoyne this file has very similar contents to bin/webpack-dev-server, except for a different runner. What is its purpose?

Reply:
It builds the javascript. See https://github.com/shakacode/shakapacker#development